unlock-app/src/middlewares/walletMiddleware.js

```diff
@@ -225,6 +230,30 @@ const walletMiddleware = config => {
               }
             }
           )
+        } else if (action.type === SIGN_METADATA_REQUEST) {
+          const { address, owner, timestamp } = action
+          // Usage from locksmith tests for metadataController
```

Comment:
nit: Clarify?

Reply:
Any suggestions on how to? Since we don't have types for this stuff I just want to leave a breadcrumb trail for future us in the event we have to debug this.